src/orchestration/Orchestration.ts

```diff
@@ -41,11 +41,7 @@ interface TelemetriesFunctor {
 
 type Telemetry = string | (string | object)[];
 
-export enum PAGE_ID_FORMAT {
-    PATH = 'PATH',
-    HASH = 'HASH',
-    PATH_AND_HASH = 'PATH_AND_HASH'
-}
```

Comment:
issue: This enum can still be used to reference the page ID formats so we don't have to write out the string literals. The issue is related to the type of `PartialConfig.pageIdFormat` and `Config.pageIdFormat`. So if you change the type of those values to `PageIdFormat` I think the problem is solved.

Reply:
observation: At some point we should export all the types used in `PartialConfig` from `index.ts`. This includes `PAGE_ID_FORMAT`.  This will allow the types to be used by the applications installing the web client.

We could do it in this PR, or create a separate PR for it later.

Reply:
(1) Moved the enums in `Orchestration.ts` to `constants.ts` in the same directory
(2) Added 4 types (Telemetry, PageIdFormat, PartialCookieAttribute, and ClientBuilder) to `index.ts` to export.